bin/submission_new.py

```diff
@@ -434,6 +432,8 @@ def create_xml(self, output_dir):
 		submission = ET.Element('Submission')
 		# Description block (common across all submissions)
 		description = ET.SubElement(submission, 'Description')
+		if "Specified_Release_Date" in self.submission_config:
+			release_date = etree.SubElement(description, "Hold", release_date=self.submission_config["Specified_Release_Date"])
```

Comment:
Thank you for adding this!

Reply:
Can you add Specified_Release_Date to the submission_config.yaml file?  Just leave it blank ("")
And modify this to say if it's not blank or "Not Provided", then add the release_date element.

Reply:
Modified submission_config.yaml and added an extra condition in this conditional statement.